app/routes.js

```diff
@@ -1,8 +1,10 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const weets = require('./controllers/weets');
 
 exports.init = app => {
   app.get('/health', healthCheck);
+  app.post('/weet', weets.postNewWeet);
```

Comment:
a path is not necessary because it must be tested by console

Reply:
path removed